llvm/test/CodeGen/NVPTX/bf16-instructions.ll

```diff
@@ -32,8 +31,8 @@ define bfloat @test_fadd(bfloat %0, bfloat %1) {
 ; SM70-NEXT:    setp.nan.f32 %p1, %r5, %r5;
 ; SM70-NEXT:    or.b32 %r9, %r5, 4194304;
 ; SM70-NEXT:    selp.b32 %r10, %r9, %r8, %p1;
-; SM70-NEXT:    mov.b32 {_, %rs1}, %r10;
-; SM70-NEXT:    st.param.b16 [func_retval0], %rs1;
+; SM70-NEXT:    shr.u32 %r11, %r10, 16;
```

Comment:
Looks like we've lost the partially discarding `mov`. Was that intentional? Not sure if it makes any difference for SASS.

Reply:
I think this is an acceptable side effect of moving to conventional store nodes which TRUNCATE nodes then get folded into, preventing the Pat that would generate the mov. I don't see much of a significant impact on the SASS https://cuda.godbolt.org/z/djfhKcxMW. We could consider canonicalizing all shifts by constant multiples of 8 to `prmt` if this turns out to be an issue.